source/includes/_imageannotation.mdown

```diff
@@ -874,3 +878,121 @@ The `response` field, which is part of the callback POST request and permanently
 <aside class="notice">
 See the <a href="#callbacks">Callback section</a> for more details about callbacks.
 </aside>
+
+## Layers
+> Example layers param with all layer types in use
+
+```json
+{
+    "lines": [
+        {
+            "vertices": [
+                {
+                    "y": 323,
+                    "x": 414
+                },
+                {
+                    "y": 164,
+                    "x": 616
+                },
+                {
+                    "y": 334,
+                    "x": 776
+                }
+            ],
+            "label" : "crosswalk"
+        }
+    ],
+    "polygons": [
+        {
+            "vertices": [
+                {
+                    "y": 145,
+                    "x": 356
+                },
+                {
+                    "y": 103,
+                    "x": 502
+                },
+                {
+                    "y": 264,
+                    "x": 482
+                }
+            ],
+            "label": "building"
+        }
+    ],
+    "boxes": [
+        {
+            "label": "car",
+            "height": 97,
+            "width": 147,
+            "top": 229,
+            "left": 300
+        }
+    ],
+    "cuboids": [
+        {
+            "vertices" : [
+                {
+                    "description" : "face-topleft",
+                    "y" : 219.0,
+                    "x" : 137.0
+                },
+                {
+                    "description" : "face-bottomleft",
+                    "y" : 318.0,
+                    "x" : 137.0
+                },
+                {
+                    "description" : "face-topright",
+                    "y" : 219.0,
+                    "x" : 245.0
+                },
+                {
+                    "description" : "face-bottomright",
+                    "y" : 318.0,
+                    "x" : 245.0
+                },
+                {
+                    "description" : "side-topcorner",
+                    "y" : 165.0,
+                    "x" : 316.0
+                },
+                {
+                    "description" : "side-bottomcorner",
+                    "y" : 264.0,
+                    "x" : 316.0
+                }
+            ],
+            "label" : "car"
+        }
+    ]
+}
+```
+
+“Layers” can be used to specify existing (read-only) boxes, lines, polygons, and/or cuboids to be pre-drawn on an image. For instance, you could specify boxes around the cars of an image, for a task that requires drawing polygons around the currently boxed cars.
+
```

Comment:
I would add some info about use cases:

"Suppose you wanted someone to draw boxes around all cars which you hadn't already recognized..."

I would also add something about how to easily integrate this using existing Scale tasks:

"All of these input formats have the same formatting as our responses, so passing one Scale tasks' data to another Scale task is as easy has: {some code example}"

Reply:
Done! I added examples in node, ruby and python - if you agree with them conceptually I'll download the SDKs, test the code and make any necessary modifications if there is any issue (e.g. I'm not 100% sure the Python and Ruby SDKs support nested params right now).